website/docs/events/m2.md

```diff
@@ -0,0 +1,127 @@
+
+In spite of all of this, a large number of submissions which receive high peer-grades are not of the quality to be acceptable as a part of course material in any sense.
+
+To counter the last of these issues, submissions which have received a good peer-review (possibly after resubmission) may be subjected to a review by a vetted reviewer (the instructor) in SmileyTutor.
+An exercise or drill which has been deemed of high quality following these reviews can be considered for inclusion in current or future course material.
+
+These multiple uses of authoring, peer-review and vetted review are not seen as fundamental or obvious parts of OpenEdu.
+
+However, OpenEdu has at its core a storage mechanism which can be used to store vetted student-submissions into SmileyTutor.
+This leads to the second important link between the two systems: Storing SmileyTutor student-submissions in OpenEdu.
+
+This second link has not been implemented yet and was therefore not discussed further at M2.
+
+### Migration issues: The CCAS course notes
+
+The meeting was presented with a fairly large number of issues involved in migrating the CCAS course notes into the current OpenEdu system.
+Chief among them being the interplay between Markdown and LaTeX within GitHub’s Markdown viewer.
+
+A short demonstration was given on the conversion workflow, which utilised a reasonable automatic LaTeX-to-Markdown converter, but had to be both pre- and post-processed by some fairly sophisticated python scripts.
+
+In addition, almost all of the images in the notes were being generated from R scripts.
+As such functionality is obviously not reproducible with base Markdown, the scripts for the images were run locally and the resulting images could be referenced by Markdown.
+
+Some comments followed on the difficulties and nature of repurposing content within education.
+
+### Rewarding students
+
+Students are commonly rewarded using grades but can also be rewarded for work using other currencies such as smiley faces or points.
+
+The SmileyCoin cryptocurrency is well established as a reward mechanism in the SmileyTutor and will be included in OpenEdu.
+A real-world course can use either system and students will be able to redeem [SmileyCoin](https://smileyco.in/#/) for appropriate answers or submissions as the case may be.
+The SmileyCoin is an example of transferring technology from SmileyTutor to OpenEdu.
+
+Within SmileyTutor an elaborate item allocation algorithm has been established for assigning the next drill to a student and a correspondingly elaborate weighting mechanism for grading has been established to emphasise learning-by-doing rather than examining students.
+
+The approaches taken by SmileyTutor and OpenEdu are fundamentally different.
+
+### Using OpenEdu as content storage: Course notes
+
+Once course notes have been set up in OpenEdu along with a compiled PDF file, it is trivial to insert a link to the PDF file from SmileyTutor.
```

Comment:
```suggestion
Once course notes have been set up in OpenEdu along with a compiled PDF file, it is trivial to insert a link to the PDF file from SmileyTutor.
```
Ideally we wouldn't store binary files (such as PDFs) on GitHub, but rather implement some sort of a builder to create it, host it on a given platform, and then point SmileyTutor to that generated PDF instead.